[Bug]: Compilation failed on Windows

### Summary

There are many errors when compiling on Windows platform

### Affected component

runtime/daemon

### Severity

S2 - degraded behavior

### Current behavior

Exit code is non zero

### Expected behavior

Compile successfully.

### Steps to reproduce

```bash
cargo build --release
```

### Impact

Yes

### Logs / stack traces

```text
Compiling zeroclaw v0.1.7 (F:\rust\zeroclaw) 
warning: unused import: `std::io::ErrorKind` 
 --> src\update.rs:8:5 
 | 
8 | use std::io::ErrorKind; 
 | ^^^^^^^^^^^^^^^^^^ 
 | 
 = note: `#[warn(unused_imports)]` (part of `#[warn(unused)]`) on by default 
 
error[E0658]: use of unstable library feature `windows_by_handle` 
 --> src\security\file_link_guard.rs:20:24 
 | 
20 | u64::from(metadata.number_of_links()) 
 | ^^^^^^^^^^^^^^^ 
 | 
 = note: see issue #63010 <https://github.com/rust-lang/rust/issues/63010> for more information 
 
error[E0277]: the trait bound `u64: std::convert::From<std::option::Option<u32>>` is not satisfied 
 --> src\security\file_link_guard.rs:20:5 
 | 
20 | u64::from(metadata.number_of_links()) 
 | ^^^ the trait `std::convert::From<std::option::Option<u32>>` is not implemented for `u64` 
 | 
 = help: the following other types implement trait `std::convert::From<T>`: 
 `u64` implements `std::convert::From<aws_lc_rs::endian::BigEndian<u64>>` 
 `u64` implements `std::convert::From<aws_lc_rs::endian::LittleEndian<u64>>` 
 `u64` implements `std::convert::From<bool>` 
 `u64` implements `std::convert::From<char>` 
 `u64` implements `std::convert::From<cranelift_codegen::ir::immediates::Uimm32>` 
 `u64` implements `std::convert::From<cranelift_codegen::ir::immediates::Uimm64>` 
 `u64` implements `std::convert::From<std::ascii::Char>` 
 `u64` implements `std::convert::From<u16>` 
 and 5 others 
 
Some errors have detailed explanations: E0277, E0658. 
For more information about an error, try `rustc --explain E0277`. 
warning: `zeroclaw` (lib) generated 1 warning 
error: could not compile `zeroclaw` (lib) due to 2 previous errors; 1 warning emitted
```

### ZeroClaw version

v0.1.7

### Rust version

1.93.1

### Operating system

Windows 11

### Regression?

Unknown

### Pre-flight checks

- [x] I reproduced this on the latest main branch or latest release.
- [x] I redacted secrets/tokens from logs.
- [x] I removed personal identifiers and replaced identity-specific data with neutral placeholders.

--- a/src/security/file_link_guard.rs
+++ b/src/security/file_link_guard.rs
@@ -15,9 +15,11 @@ fn link_count(metadata: &Metadata) -> u64 {
 }
 
 #[cfg(windows)]
-fn link_count(metadata: &Metadata) -> u64 {
-    use std::os::windows::fs::MetadataExt;
-    u64::from(metadata.number_of_links())
+fn link_count(_metadata: &Metadata) -> u64 {
+    // Rust stable does not currently expose a portable, stable Windows hard-link
+    // count API on `std::fs::Metadata`. Returning 1 avoids false positive blocks
+    // and keeps Windows builds stable until a supported API is available.
+    1
 }
 
 #[cfg(not(any(unix, windows)))]

